### DISCUSSION TITLE
Github API

### DISCUSSION THREAD

**Discussion by shawnthompson** [gc-da11yn/gc-da11yn.github.io#70] [Development (Open discussions)]
What about creating HTML pages using the [Github GraphQL API](https://docs.github.com/en/graphql/overview/about-the-graphql-api) or the [REST API](https://docs.github.com/en/rest) to list the open issues and the discussions on the site?

* [REST API - Issues](https://docs.github.com/en/rest/reference/issues)
* [Using the GraphQL API for Discussions](https://docs.github.com/en/graphql/guides/using-the-graphql-api-for-discussions)

I know @duboisp did this in WET-BOEW.

**Comment by duboisp** [comment]
It is a feature that is already available because we are leveraging GCWeb for gc-da11yn website.

Here a working example that use GitHub RestFull API : https://wet-boew.github.io/GCWeb/components/wb-jsonmanager/jsonmanager-en.html
The GitHub GraphQL API usually require an authentication witch is not implemented yet in the JSON manager plugin.

**Comment by shawnthompson** [comment]
Awesome. Is there any thoughts as to when the authentication will be added to WET?
